app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
Note that this renders `messages` instead of `message` like the visits API does. I think an array works better instead of rendering error messages individually. We should choose one or the other and keep it consistent.

Or we could consider using [ActiveModel::Serializer](https://github.com/rails-api/active_model_serializers) for serialisation. They were part of the `rails-api` project that eventually got merged into Rails

I had a good experience using it in the past, makes it easier to be consistent when writing an API. Decisions like how to render error are all handled by the serialiser which depends on what adapter you are using in the latest version ie json-api, custom one, etc. If we don't want to follow a particular standard we could then have a generic moj adapter to have consistent API's. What do you think?